R/cookies.R

```diff
@@ -1,10 +1,55 @@
 #' dfe_cookie_script
 #'
+#' Calls in JavaScript dependencies to the shiny app. Function should be placed
+#' in the ui.R script.
+#'
+#' @name cookies
 #' @return shiny::tags$head()
 #' @export
 #'
+#' @family cookies
 #' @examples
-#' dfe_cookie_script()
+#' # This example shows how to use the full family of cookie functions together
+#' # This will be in your global.R script =====================================
+#'
+#' library(shiny)
+#' library(shinyjs)
+#' library(dfeshiny)
+#' ga_key <- "ABCDE12345"
+#'
+#' # This will be what is in your ui.R script =================================
+#'
+#' ui <- fluidPage(
+#'   # Place these lines above your header ------------------------------------
+#'   dfe_cookie_script(),
+#'   useShinyjs(),
+#'   cookie_banner_ui(name = "My DfE R-Shiny data dashboard"),
+#'
+#'   # Place the cookies panel under the header but in the main content -------
+#'   cookies_panel_ui(google_analytics_key = ga_key)
+#' )
+#'
+#' # This will be in your server.R file =======================================
+#'
+#' server <- function(input, output, session) {
+#'   # Server logic for the pop up banner, can be placed anywhere in server.R -
+#'   output$cookie_status <- dfeshiny::cookie_banner_server(
+#'     input_cookies = reactive(input$cookies),
+#'     google_analytics_key = ga_key
+#'   )
+#'
+#'   # Server logic for the panel, can be placed anywhere in server.R ---------
+#'   cookies_panel_server(
+#'     input_cookies = reactive(input$cookies),
+#'     google_analytics_key = ga_key
+#'   )
+#' }
+#'
+#' # How to run the minimal app give in this example ==========================
```

Comment:
is this meant to be "given" rather than "give"?

Reply:
Yep! Good spot, fixed now